facilitator/src/bin/facilitator.rs

```diff
@@ -793,6 +841,76 @@ fn aggregate(sub_matches: &ArgMatches) -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+fn lint_manifest(sub_matches: &ArgMatches) -> Result<(), anyhow::Error> {
+    let manifest_base_url = sub_matches.value_of("manifest-base-url");
+    let manifest_file_reader = match sub_matches.value_of("manifest-path") {
+        Some(path) => Some(File::open(path).context("failed to open manifest file")?),
+        None => None,
+    };
+
+    let manifest_kind = ManifestKind::from_str(
```

Comment:
I don't think this needs changing but wanted to point out an alternate syntax that's sometimes nice:

```
let kind: ManifestKind = sub_matches.value_of("manifest-kind").context("manifest-kind is required")?.parse()?;
```

That gets you a couple of things:
 - The type is annotated on the variable
 - It's a linear chain of calls rather than nested parens.

Reply:
This does read more nicely, good tip!